viper/modules/virustotal.py

```diff
@@ -355,11 +368,17 @@ def run(self):
                 if eid:
                     if __sessions__.is_attached_misp(quiet=True):
                         if __sessions__.current.misp_event.event.id != int(eid):
-                            self.log('warning', 'You opened a sample related to a MISP event different than the one you are currently connected to: {}.'.format(eid))
+                            self.log('warning',
+                                     'You opened a sample related to a MISP event different than the one you are '
+                                     'currently connected to: {}.'.format(
+                                         eid))
                         else:
                             self.log('success', 'You opened a sample related to the current MISP event.')
                     else:
-                        self.log('warning', 'This samples is linked to the MISP event {eid}. You may want to run misp pull {eid}'.format(eid=eid))
+                        self.log('warning',
```

Comment:
How do you feel about a max character per line line limit in general? I know there are laods and loads of discussion (maybe even flamewars) on this topic. 

My opinion is that having a limit (e.g. 100/120) is useful - but of course exceptions can we easily be made on a per file or per line basis.